backend/routes/indexer/route.go

```diff
@@ -72,6 +72,7 @@ const (
 
 var eventProcessors = map[string](func(IndexerEvent)){
 	newDayEvent:             processNewDayEvent,
+	colorAddedEvent:         processColorAddedEvent,
```

Comment:
The `colorAddedEvent` string isn't defined. It should be `0x0004a301e4d01f413a1d4d0460c4ba976e23392f49126d90f5bd45de7dd7dbeb`